### PR TITLE
3564: Incorporate welsh translations sent as of May 27th.

### PR DESCRIPTION
**Pull request by Veda**
Authors: @veda

### CODE REVIEW

File: config/locales/cy.yml [hunk 1]
```diff
@@ -22,11 +22,11 @@ cy:
...
     confirm_your_identity: cadarnhau-eich-hunaniaeth
     confirmation: cadarnhad
-    failed_registration: failed-registration-cy
-    failed_sign_in: failed-sign-in-cy
-    other_ways_to_access_service: other-ways-to-access-service-cy
-    forgot_company: forgot-company-cy
-    response_processing: response-processing-cy
+    failed_registration: cofrestru-wedi-methu
+    failed_sign_in: mewngofnodi-wedi-methu
+    other_ways_to_access_service: ffyrdd-eraill-i-gael-mynediad-i'r-gwasanaeth
```
Comment (robinwhittleton) [inline config/locales/cy.yml]: I guess apostrophes in paths are OK?

**Comment by robinwhittleton** [comment]
There are a few places in the text with '`' instead of `’`, happy to merge once they’re fixed.

File: config/locales/cy.yml (hunk 1 above)
Comment (Veda) [inline config/locales/cy.yml]: Normal ones are ok, tests are working alright.

**Comment by Veda** [comment]
@robinwhittleton Done with updating to curly apostrophes. Added a missing test in addition.

**Pull request merged**